packages/media/src/webrtc/VegaRtcManager/index.ts

```diff
@@ -124,10 +126,17 @@ export default class VegaRtcManager implements RtcManager {
         this._micAnalyser = null;
         this._micAnalyserDebugger = null;
 
+        const handlerName = getHandler();
+
+        if (handlerName === "Safari17" && !deviceHandlerFactory) {
```

Comment:
I wonder if we should include a feature check here and roll out gradually?
This will affect a lot of users immediately, potentially on versions of Safari I can't test as I only have access to 17.4 and 18.1.1

Reply:
Yes, please put it behind a feature flag.